### ISSUE TITLE
Property 'xx' does not exist on type 'ElementAttrs'

### ISSUE THREAD

**Issue by linjhon** [vuejs/vue#12620]
### Version
2.7.3

### Reproduction link
[github.com](https://github.com/linjhon/vue-project-2.7)

### Steps to reproduce
1. clone reproduction repo
2. pnpm install
3. npm run typecheck

### What is expected?
Don't show error

### What is actually happening?
1. src/components/icons/IconTooling.vue:5:5

```html
 <svg
  xmlns="http://www.w3.org/2000/svg"
  xmlns:xlink="http://www.w3.org/1999/xlink"
 >
  //...
 </svg>
```
```sh
error TS2322: Type '{ xmlns: string; "xmlns:xlink": string; ariaHidden: string; "aria-hidden": "true"; role: string; class: string; width: string; height: string; preserveAspectRatio: string; viewBox: string; }' is not assignable to type 'ElementAttrs<SVGAttributes>'.
  Property 'xmlns:xlink' does not exist on type 'ElementAttrs<SVGAttributes>'.
```

2. src/App.vue:14:9

```html
<img
  alt="Vue logo"
  class="logo"
  src="@/assets/logo.svg"
  width="125"
  height="125"
  aria-disabled="true"
/>
```
```sh
error TS2322: Type '{ alt: string; class: string; src: string; width: string; height: string; ariaDisabled: string; "aria-disabled": "true"; }' is not assignable to type 'ElementAttrs<ImgHTMLAttributes>'.
  Property 'ariaDisabled' does not exist on type 'ElementAttrs<ImgHTMLAttributes>'. Did you mean ''aria-disabled''?
```

**Comment by yyx990803** [comment]
/cc @johnsoncodehk

The JSX type should be consistent with v3 - does this only happen when using `vue-tsc` with Vue 2?

**Comment by johnsoncodehk** [comment]
This is fixed by https://github.com/johnsoncodehk/volar/pull/1533 and released in Volar v0.38.3, please update the Volar extension version.
